plugins/no-override.js

```diff
@@ -18,22 +15,30 @@ module.exports = stylelint.createPlugin(ruleName, (enabled, options = {}) => {
   const primerMeta = requirePrimerFile('dist/meta.json')
   const availableBundles = Object.keys(primerMeta.bundles)
 
-  const immutableSelectors = new Set()
-  const immutableClassSelectors = new Set()
+  const immutableSelectors = new Map()
+  const immutableClassSelectors = new Map()
   for (const bundle of bundles) {
     if (!availableBundles.includes(bundle)) {
       continue
     }
     const stats = requirePrimerFile(`dist/stats/${bundle}.json`)
-    for (const selector of stats.selectors.values) {
-      immutableSelectors.add(selector)
+    const selectors = stats.selectors.values.filter(selector => !!CLASS_PATTERN.test(selector))
```

Comment:
```suggestion
    const selectors = stats.selectors.values.filter(selector => CLASS_PATTERN.test(selector))
```
Doesn't `.test()` always return a boolean? Why do we need `!!` here?